jcommune-view/jcommune-web-view/src/main/webapp/WEB-INF/jsp/topic/postList.jsp

```diff
@@ -284,7 +284,7 @@
       </div>
              
                    <span class="thumbnail post-userinfo-avatal wraptocenter">
-                        <img src="${pageContext.request.contextPath}/users/${post.userCreated.id}/avatar" alt=""/>
```

Comment:
I can't know. This part of code is not mine, it was being here before I changed the line. My changes only affect version of avatar.